src/components/__tests__/MutationTables.test.tsx

```diff
@@ -0,0 +1,125 @@
+const locationSelector: LocationSelector = {
+  region: undefined,
+  country: 'Iceland',
+  division: undefined,
+};
+
+describe('SingleMutationsTable', () => {
+  const mutations: MutationObject[] = [
+    {
+      name: 'BF.7.14',
+      query: { nextcladePangoLineage: 'BF.7*' },
+      variantCout: 500,
+      aaMutations: [
+        { proportion: 0.09, mutation: 'ORF8:G77C', count: 2 },
+        { proportion: 0.99, mutation: 'ORF8:G77A', count: 3 },
+      ],
+      nucMutations: [
+        { proportion: 0.09, mutation: 'C20404T', count: 4 },
+        { proportion: 0.99, mutation: 'T670G', count: 5 },
+      ],
+    },
+    {
+      name: 'BA.5.2.48',
+      query: { nextcladePangoLineage: 'BA.5.2*' },
+      variantCout: 1000,
+      aaMutations: [
+        { proportion: 0.05, mutation: 'ORF8:A65V', count: 2 },
+        { proportion: 0.06, mutation: 'ORF8:A51V', count: 3 },
+      ],
+      nucMutations: [
+        { proportion: 0.07, mutation: 'T670G', count: 4 },
+        { proportion: 0.08, mutation: 'C3264T', count: 5 },
+      ],
+    },
+  ];
+
+  it('the table is rendered', () => {
+    render(
+      <BrowserRouter>
+        <SingleMutationsTable
+          mutations={mutations}
+          locationSelector={locationSelector}
+          mutationType={mutationTypeAa}
+          dateRangeSelector={dateRangeSelector}
+        />
+      </BrowserRouter>
+    );
+
+    const table = screen.getByTestId('single-mutation-table');
+    const rows = screen.getAllByText((_, element) => element?.tagName.toLowerCase() === 'tr');
+    const cells = screen.getAllByText((_, element) => element?.tagName.toLowerCase() === 'td');
+
+    expect(table).toBeInTheDocument();
+    expect(rows.length).toBe(3); // 2 mutations plus one header
+    expect(cells.length).toBe(15); // 3 rows * 5 columns
+    expect(cells[6].textContent).toBe('ORF8:G77A'); // > 90 %
+    expect(cells[9].textContent).toBe('ORF8:G77C'); // < 5%, < 30%
+  });
+});
+
+describe('BaselineMutationsTable', () => {
+  const baselineData: BaselineData = {
+    aaMutations: [
+      { proportion: 0.09, mutation: 'ORF8:G77C', count: 2 },
+      { proportion: 0.99, mutation: 'ORF8:G78A', count: 3 },
+    ],
+    nucMutations: [
+      { proportion: 0.09, mutation: 'C20404T', count: 4 },
+      { proportion: 0.99, mutation: 'T670G', count: 5 },
+    ],
+  };
+
+  const mutations: MutationObject[] = [
+    {
+      name: 'BF.7.14',
+      query: { nextcladePangoLineage: 'BF.7*' },
+      variantCout: 500,
+      aaMutations: [
+        { proportion: 0.01, mutation: 'ORF8:G77C', count: 2 },
+        { proportion: 0.94, mutation: 'ORF8:G90A', count: 3 },
+        { proportion: 0.05, mutation: 'ORF8:G78A', count: 3 },
+      ],
+      nucMutations: [
+        { proportion: 0.051, mutation: 'C20404T', count: 4 },
+        { proportion: 0.091, mutation: 'T670G', count: 5 },
+      ],
+    },
+  ];
+
+  it('the table is rendered', () => {
+    render(
+      <BrowserRouter>
```

Comment:
Doesn't change much here, but I think it's better to use `MemoryRouter` in tests: https://v5.reactrouter.com/web/guides/testing